app/src/controllers/userController.ts

```diff
@@ -0,0 +1,60 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+export class UserController {
+
+    public async index (req: Request, res: Response) {
+        try {
+            const users = await User.find();
+            res.json(users);
+        } catch(err) {
+            res.send(err);
+        }
+    }
+
+    public async create (req: Request, res: Response) {
+        try {
+            let newUser = new User(req.body);
```

Comment:
why req.body?

Reply:
@khaydarov because the parameters that used to update the user are in the request body? where else should the parameters be?

Reply:
It would be better to pass arguments to the "User" Model explicitly. Request Body may contain unused/taint fields so that it is needed to validate and filtered before creating the concrete Model